language/Pattern-Matching.md

```diff
@@ -33,6 +33,7 @@ The following pattern types are supported:
 - Record patterns
 - Named patterns
 - Guards
+- Pattern guards
```

Comment:
Is there any place where patterns are allowed but guards are not? View patterns in Haskell even desugar to guards, so I would lean towards saying they are _at least_ intimately related concepts.